docs/howto/planning_splinter_deployment.md

```diff
@@ -0,0 +1,157 @@
+ this guide explains what you need to know before you start.
+
+Splinter works with a wide variety of deployment patterns. This guide summarizes
+how to use Docker containers, a basic Kubernetes pod, and Amazon Elastic
+Kubernetes Services (Amazon EKS).
+
+**Contents**
+<br>
+[Overview](#bookmark=id.ls4meksnxna)
+<br>
+[Splinter Architecture](#bookmark=id.qwe1rp9mdlx5)
+<br>
+[Network Configuration](#bookmark=id.brrcajnirna1)
+<br>
+[Application Pattern](#bookmark=id.w243im9kbun8)
+<br>
+[Docker Deployment](#bookmark=id.i8nr28r242j4)
+<br>
+[Basic Kubernetes Deployment](#bookmark=id.4gmpe3v077wl)
+<br>
+[Amazon EKS Deployment](#bookmark=id.ism06ku1ut2h)
+
+# Overview
+
+A Splinter deployment usually has the following items:
+
+* A system (called a _node_) that runs the Splinter daemon (splinterd) and
+  associated components. Splinter includes a REST API for applications to
+  communicate with splinterd.
+
+* A server-side application with a local database for storing persistent data,
+  such as transaction-based changes from Splinter's _state delta export_
+  function.
+
+*   A web UI (brower-based user interface) for interacting with the application.
+
+![Splinter deployment overview](../images/splinter-deployment-overview.png "Splinter deployment overview")
+
+# Splinter Architecture
+
+Splinter is a privacy-focused platform for distributed applications that provides a blockchain-inspired networking environment for communication and transactions between organizations.
+
+Splinter provides a networking layer and services for running smart contracts so
+that applications can build upon this functionality. A Splinter network allows
+both network-wide shared state and private communication between two or more
+participants (Splinter nodes), all managed with consensus. Splinter ensures
+privacy for multi-node conversations, as well as the state shared by these
+nodes, by using **circuits** and **services**.
+
+* A **circuit** is a virtual network within the broader Splinter network that
+  safely and securely enforces privacy scope boundaries. Connections are
+  dynamically constructed between nodes as circuits are created.
+
+* A **service** is an endpoint within a circuit that sends and receives private
+  messages. All nodes on a circuit run the same set of services.
+
+* **Shared state** (a database updated by smart contracts) is visible only to
+  the services within a circuit.
+
+For more information about Splinter, see the [Splinter
+documentation](https://github.com/Cargill/splinter-docs/blob/master/docs/index.md).
+
+# Network Configuration
+
+The Splinter daemon, splinterd, communicates with other nodes on the network and
+provides a REST API to expose Splinter functions and data to your application.
+The Splinter daemon uses the following ports:
+
+* **8044**: Network endpoint for node-to-node communication
+
+* **8085**: Application communication (encrypted with SSL) via the splinterd
```

Comment:
This is not actually true.  The REST API for splinterd is served over standard HTTP.  If the deployment requires HTTPS, the current thinking is that there should be a proxy server for the HTTPS requests.

Reply:
Fixed.